### PR TITLE
Switch to pyup.io

### PR DESCRIPTION
**Pull request by stephendonner**
DO NOT MERGE (yet), please :-)

This addresses the first part of https://github.com/mozilla-services/go-bouncer/issues/65

I need to figure out why these four tests fail (likely due to upgraded "requests" library?):

[gw2] FAILED tests/e2e/tests/test_smoketests.py::TestSmokeTests::test_verify_firefox_aliases_redirect_to_correct_products[linux] 
[gw5] FAILED tests/e2e/tests/test_smoketests.py::TestSmokeTests::test_verify_releng_aliases_match_what_we_expect 
[gw6] FAILED tests/e2e/tests/test_smoketests.py::TestSmokeTests::test_verify_firefox_aliases_redirect_to_correct_products[osx] 
[gw4] FAILED tests/e2e/tests/test_smoketests.py::TestSmokeTests::test_verify_firefox_aliases_redirect_to_correct_products[win] 

Full output here: http://pastebin.com/DaYDsrZE

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -4,14 +4,9 @@ skipsdist = true
 
 [testenv]
 passenv = PYTEST_ADDOPTS
-deps =
-    beautifulsoup4==4.5.1
-    pytest~=3.0.0
-    pytest-base-url==1.1.0
-    pytest-xdist==1.15.0
-    requests==2.11.1
+deps = -rrequirements/tests.txt
```
Comment (m8ttyB) [inline tox.ini]: Is this correct, the double r's? `rrequirements/`

Comment (m8ttyB) [inline tox.ini]: Ah, I see it in https://github.com/mozilla/PyPOM/commit/ad21988d0364843808c07f3aaa656c6c16b971b2

Comment (davehunt) [inline tox.ini]: Yes, this is Tox's syntax, and is documented [here](http://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt).

File: requirements/tests.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+beautifulsoup4==4.5.1
+pytest==3.0.5
+pytest-base-url==1.2.0
+pytest-xdist==1.15.0
+requests==2.12.4
```
Comment (davehunt) [inline requirements/tests.txt]: Can we avoid updating dependencies at the same time, as this is exactly what pyup.io will do anyway.

Comment (stephendonner) [inline requirements/tests.txt]: @davehunt done!

File: requirements/tests.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+beautifulsoup4==4.5.1
+pytest==3.0.0
```
Comment (davehunt) [inline requirements/tests.txt]: This is actually pinning to an earlier version.

**Comment by davehunt** [comment]
Tests are passing for me, but we're expecting a CircleCI status... Looking at https://circleci.com/gh/mozilla-services/go-bouncer there hasn't been a build for over a month. @oremj can you help us to investigate?

**Comment by oremj** [comment]
Looks like the webhook was removed, somehow. It should work now.

**Pull request merged**